README.md

```diff
@@ -47,9 +47,28 @@ For configuring Grafana, navigate to http://localhost:8000/dashboards. Here, you
 
 To view the list of alerts in Prometheus, visit http://localhost:9090/alerts. For checking the list of fired alerts in Alertmanager, go to http://localhost:9093/#/alerts.
 
+## Update CSM module ABI
+
+Execute the command in the CSM module repository
```

Comment:
make we need to add repo link, it is not clear what do we mean here by CSM module repository